docusaurus/docs/iOS/04-ui-components/09-utility/03-avatars.mdx

```diff
@@ -15,4 +15,4 @@ var body: some View {
 }
 ```
 
-The view has a circled shape. If that does not fit your UI requirements, you can easily build your own view, by either using the native `AsyncImage`, or `Nuke`'s `LazyImage`.
+The view has a circled shape. If that does not fit your UI requirements, you can easily build your own view, by either using the native `StreamLazyImage`.
```

Comment:
"by either using the native `StreamLazyImage`", probably the "either" is extra now.